src/listeners.js

```diff
@@ -32,6 +33,12 @@ class Listener{
                 this._getListenersFromUISchema(value, path + '/' + key, acc)
             }.bind(this))
         }
+        // check list items for fields with listeners
+        if (ui_schema["item"] && ui_schema["item"]["fields"]){
+            _.each(ui_schema["item"]["fields"], function (value, key) {
```

Comment:
In general, can we try to replace the `lodash` stuff with native ES6 where possible here?

Reply:
yeah - in favor of changing as little as possible - I will change the ones that I changed, but will leave the existing ones in place. We are hotfixing this feature, so I don't want to mess with too much.

Reply:
I swapped it in one place, but here fields is actually a dict, which I suppose I could cycle through the keys, but that feels harder to read? I am going to leave like this for now, since we don't get to remove lodash stuff because we are using it in other methods I would rather not touch since this is a hotfix feature. I think there is def some tech debt for this that could be cleaned up.